docs/solr.md

```diff
@@ -50,3 +50,48 @@ Note: Limited metrics will be reported if no cores have been created.
 | workload.googleapis.com/solr.cache.insert.count      | Cumulative     | inserts     | core, cache                    | The number of inserts from a cache. |
 | workload.googleapis.com/solr.cache.lookup.count      | Cumulative     | lookups     | core, cache                    | The number of lookups from a cache. |
 | workload.googleapis.com/solr.cache.size              | Gauge          | by          | core, cache                    | The size of the cache occupied in memory. |
+
+# `solr_system` Logging Receiver
+
+## Configuration
+
+To configure a receiver for your Solr system logs, specify the following fields:
+
+| Field                 | Default                           | Description |
+| ---                   | ---                               | ---         |
+| `type`                | required                          | Must be `solr_system`. |
+| `include_paths`       | `[/var/solr/logs/solr.log]`       | A list of filesystem paths to read by tailing each file. A wild card (`*`) can be used in the paths; for example, `/var/solr/logs/*.log`.
+| `exclude_paths`       | `[]`                              | A list of filesystem path patterns to exclude from the set matched by `include_paths`.
```

Comment:
Also `wildcard_refresh_interval`? Similar to https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/apache#apache-logs

Reply:
@qingling128 added